libtsuba/include/tsuba/tsuba.h

```diff
@@ -15,6 +15,7 @@
 namespace tsuba {
 
 class RDGHandleImpl;
+class RDGManifest;
```

Comment:
Just `#include "tsuba/RDGManifest.h"` rather than using this forward declaration. Users of this header will have to know about that type anyway.

Reply:
@tylershunt There is a circular inclusion issue between `tsuba/tsuba.h` and `tsuba/RDGHandle.h`, `tsuba/RDGManifest.h`. So I have to remain a declaration here.

Reply:
Ah I see. Annoying; that is a mess but carry on.